### PR TITLE
Add Array#last limit til

### PR DESCRIPTION
**Pull request by cored**
`Array#last` takes an argument to return the amount of elements from the tail of a list.

### CODE REVIEW

File: ruby/array-last-limit.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Returning a fixed amount of items from the tail of a list
```
Comment (mike-burns) [inline ruby/array-last-limit.md]: s/amount/number

File: ruby/array-last-limit.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+[`Array#last`][last docs] takes an argument to limit the number of items to return 
+from the tail of a list.
+
+```ruby
+> [1,2,3,4,5].last(2)
+# = > [4, 5]
+```
+
+[last docs](http://www.ruby-doc.org/core-2.2.0/Array.html#method-i-last)
```
Comment (JoelQ) [inline ruby/array-last-limit.md]: The syntax for footnote links is:

```md
[last docs]: http://www.ruby-doc.org/core-2.2.0/Array.html#method-i-last
```

**Comment by JoelQ** [comment]
Thanks for sharing @cored ! I've merged this in as 1bfec3347e3097fcd19e0fe5bcf05ea6bba4ae48

I'd like to credit you when we tweet this, what is your twitter handle?

**Comment by cored** [comment]
Thanks, cor3d

**Pull request closed without merging**